index.html

```diff
@@ -8,51 +8,58 @@
     <title>Keenan Botes personal portfolio </title>
 </head>
 <body>
-    <div class="container">
+    <main class="container">
```

Comment:
Kindly also note that the mobile design should also resemble the Figma design 🤗
- _Note the spaacing around the page_
- _The color of the sub-heading is different on mobile and desktop_

Your design:

![image](https://user-images.githubusercontent.com/73714615/187660231-3fa80d75-353c-4827-b496-7e7868619b7c.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187660305-f94198fc-d4c6-416e-af8a-f5054074e837.png)